aws/data_source_aws_rds_orderable_db_instance_test.go

```diff
@@ -225,11 +225,13 @@ func TestAccAWSRdsOrderableDbInstanceDataSource_supportsStorageEncryption(t *tes
 	})
 }
 
-func testAccPreCheckAWSRdsOrderableDbInstance(t *testing.T) {
+func testAccAWSRdsOrderableDbInstancePreCheck(t *testing.T) {
 	conn := testAccProvider.Meta().(*AWSClient).rdsconn
 
 	input := &rds.DescribeOrderableDBInstanceOptionsInput{
-		Engine: aws.String("mysql"),
+		Engine:          aws.String("mysql"),
+		EngineVersion:   aws.String("8.0.20"),
+		DBInstanceClass: aws.String("db.m5.xlarge"),
```

Comment:
The idea here is to reduce the time the precheck takes. It seems to take a second or two off (from ~15 to ~13 for the basic test).

Reply:
We're generally not too particular about optimizing the acceptance test times unless its a few minute gains, but as long as this doesn't become its own maintenance burden over time as the testing is done across all AWS Partitions and Regions, this is fine for now.

Reply:
👍